### PR TITLE
Update VideoIntelligence GAPIC

### PR DESCRIPTION
**Pull request by geigerj**
- All arguments configured to be keyword args for v1beta2 `annotate_video` method
- Regenerate v1beta1 with latest code generator
- Declare beta quality level

/cc @remi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ This client supports the following Google Cloud Platform services at a [Beta](#v
 * [BigQuery](#bigquery-beta) (Beta)
 * [Cloud Pub/Sub](#cloud-pubsub-beta) (Beta)
 * [Cloud Spanner API](#cloud-spanner-api-beta) (Beta)
+* [Cloud Video Intelligence API](#cloud-video-intelligence-api-alpha) (Beta)
```
Comment (quartzmo) [inline README.md]: The anchor needs to be changed to: `#cloud-video-intelligence-api-beta`

Comment (geigerj) [reply README.md]: Done.

**Comment by blowmage** [comment]
Would it be better to get these changes merged, and generate the documentation on master to verify it looks correct, before the Beta designation is added?

**Comment by blowmage** [comment]
Github is showing "This branch cannot be rebased due to conflicts". Can you rebase your PR branch against current master?

**Comment by geigerj** [comment]
@blowmage Merged in `master`.

**Comment by geigerj** [comment]
@blowmage Any other concerns before merging this PR?

**Comment by blowmage** [comment]
I still see "This branch cannot be rebased due to conflicts". I'll pull your branch and rebase it onto master manually.

**Comment by blowmage** [comment]
These commits have been moved to master manually.

**Pull request closed without merging**